projects/pgai/pgai/vectorizer/vectorizer.py

```diff
@@ -905,6 +905,21 @@ async def _do_batch(self, conn: AsyncConnection) -> int:
 
             return len(items)
 
+    @cached_property
+    def _batch_size(self) -> int:
+        """Returns the batch size for processing.
+        Documents take way longer to process than simple text rows,
+        due to download and parsing overhead.
+        So when the vectorizer is processing documents
+        we use a smaller default batch size."""
+        if self.vectorizer.config.processing.batch_size is not None:
+            return max(1, min(self.vectorizer.config.processing.batch_size, 2048))
+        else:
+            if isinstance(self.vectorizer.config.loading, UriLoading):
+                return 1
+            else:
+                return 50
```

Comment:
Can we take this opportunity to document such a default values in https://github.com/timescale/pgai/blob/main/docs/vectorizer/api-reference.md#aiprocessing_default ? 🙏